waggle-dance-integration-tests/src/test/java/com/hotels/bdp/waggledance/WaggleDanceIntegrationTest.java

```diff
@@ -134,12 +134,9 @@ private void createRemoteTable(File tableUri) throws Exception {
     File partitionAsia = new File(tableUri, "continent=Asia");
     File partitionChina = new File(partitionAsia, "country=China");
 
-    LOG
-        .info(">>>> Partitions added: {}",
-            client
-                .add_partitions(Arrays
-                    .asList(newPartition(hiveTable, Arrays.asList("Europe", "UK"), partitionUk),
-                        newPartition(hiveTable, Arrays.asList("Asia", "China"), partitionChina))));
+    LOG.info(">>>> Partitions added: {}",
```

Comment:
The formatting shouldn't have changed, can you please check your settings. Ideally in a PR we only see code that changed, not stuff being reformatted back and forth.

Reply:
It might be that this file was not formatted properly earlier and now it got formatted.